torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -354,6 +369,7 @@ def runtime_wrapper(args: list[Any]):
             try:
                 if grad_enabled:
                     torch._C._set_grad_enabled(False)
+                record_runtime_wrapper_prologue_exit(cm)
```

Comment:
Hmm at first I was going to ask you to use the context manager directly, so we don't need to worry about carefully making sure that the enter/exit line up in all the various branches.

I guess this is actually a pain to do though, since you need different global state to be toggled in the different branches, and you want to capture that inside of the recorded region